src/components/virtualizer.tsx

```diff
@@ -454,17 +471,30 @@ class Virtualizer extends React.Component<IVirtualizerProps, IState> {
         ignoredHorizontalParts={hiddenColumns}
         scrollbarSize={scrollbarSize}
       >
-        {children({
-          visibleColumnIndexes: getVisibleIndexesInsideDatalength(columnsLength, visibleColumnIndexes),
-          visibleRowIndexes: getVisibleIndexesInsideDatalength(rowsLength, visibleRowIndexes),
-          elevatedColumnIndexes,
-          elevatedRowIndexes,
-          cellHeight: this.verticalData.itemSize,
-          cellWidth: this.horizontalData.itemSize,
-        })}
```

Comment:
const virtualizerContextProps = {
  cellHeight: this.verticalData.itemSize,
            cellWidth: this.horizontalData.itemSize,
            visibleColumnIndexes: visibleAdjestedColumnIndexes,
            visibleRowIndexes: visibleAdjestedRowIndexes,
}

use it in value props of the Provider and in the children params